omod/src/main/java/org/openmrs/module/appointments/web/contract/AppointmentQuery.java

```diff
@@ -68,4 +69,12 @@ public String getServiceTypeUuid() {
     public void setServiceTypeUuid(String serviceTypeUuid) {
         this.serviceTypeUuid = serviceTypeUuid;
     }
+
+    public Boolean getIsDatelessAppointments() {
```

Comment:
Why do we need this attribute "isDatelessAppointment"? if your idea is to search for waitlisted appointment - do it by "status" not this unnecessary attribute. Unless within a particular status, you also want to search the appointments without dates, which I believe is not your requirement.

Reply:
this attribute was renamed to "withoutDates", not sure how it is showing "isDatelessAppointment". can we try refreshing the page once ?